plugins.rst

```diff
@@ -4,6 +4,13 @@ Plugins
 Plugins are an easy way to extend the features of gauge. There are
 various types of plugins that gauge currently supports.
 
+Required Plugins
+----------------
+
+-  `screenshot <https://github.com/getgauge/gauge_screenshot>`__
+
+The screenshot plugin is installed at first run automatically.
```

Comment:
Can we change this to
"The screenshot plugin is installed automatically on the first run"

?